pandas/errors/__init__.py

```diff
@@ -687,9 +687,9 @@ class LossySetitemError(Exception):
     """
     Raised when trying to do a __setitem__ on an np.ndarray that is not lossless.
 
-    Examples
+    Notes
     --------
```

Comment:
this needs to be the same length as "Notes"

Reply:
I saw it but I wasn't fast enough (before you saw it!), sorry.. doing silly errors like these.. 
I'm advancing at a painfully slow speed

Reply:
😄 no worries, I thought you'd probably seen it (or would have soon) but commented just in case